src/main/java/se/sundsvall/seabloader/integration/db/InvoiceRepository.java

```diff
@@ -33,6 +32,15 @@ public interface InvoiceRepository extends JpaRepository<InvoiceEntity, Long>, J
 	 */
 	boolean existsByInvoiceId(String invoiceId);
 
+	/**
+	 * Find by status list.
+	 *
+	 * @param statusList a List of statuses to filter on.
+	 * @return A List of invoiceIds.
+	 */
+	@Query("SELECT invoiceId FROM InvoiceEntity i where i.status in ?1")
```

Comment:
This can be done with projections instead of using native queries. See PR https://github.com/Sundsvallskommun/api-service-seab-loader/pull/21 for how its done (with projections and a interface)